static/js/tab_bar.js

```diff
@@ -38,6 +38,13 @@ function make_tab_data(filter) {
             tab_data.formatted_sub_count = '0';
             tab_data.rendered_narrow_description = "This stream does not exist or is private.";
         }
+        if (tab_data.rendered_narrow_description === "") {
+            if (page_params.is_admin) {
+                tab_data.rendered_narrow_description = "<a href=" + tab_data.stream_settings_link + ">(no description)</a>";
+            } else {
+                tab_data.rendered_narrow_description = "(no description)";
+            }
+        }
```

Comment:
I'm not happy with the HTML-escaping strategy taken here; also, there's no i18n on these various `rendred_narrow_description` elements.  It seems like we should just rework this to construct this area via an HTML template.

(In general, I don't want to add basically any code building HTML with hacky JS string concatenation, since it's had to reason about its security situation).